manifests/j/Jellyfin/JellyfinMediaPlayer/1.9.1/Jellyfin.JellyfinMediaPlayer.locale.en-US.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Jellyfin.JellyfinMediaPlayer
+PackageVersion: 1.9.1
+PackageLocale: en-US
+Publisher: Jellyfin
+PublisherUrl: https://jellyfin.org/
+PublisherSupportUrl: https://jellyfin.org/contact/
+PackageName: Jellyfin Media Player
+PackageUrl: https://github.com/jellyfin/jellyfin-media-player/
+License: GPL-2.0-or-later

```

```suggestion
License: GPL-2.0
```
I didn't see any info about later versions being supported as a license too in the repo